src/Faker/Provider/es_UY/Person.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Faker\Provider\es_UY;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @param string $ci
+     * @return string
+     */
+    public function cleanCi(string $ci): string
+    {
+        return preg_replace('/\D/', '', $ci);
+    }
+
+    /**
+     * @param string $ci
+     * @return int
+     */
+    public function validationDigit(string $ci): int
+    {
+        $ci = $this->cleanCi($ci);
+
+        $ci = str_pad($ci, 7, '0', STR_PAD_LEFT);
+
+        $a = 0;
+        $baseNumber = "2987634";
```

Comment:
What is this random looking number, please extract this to a constant.

Reply:
Ok.